firmware/openbot_nano/openbot_nano.ino

```diff
@@ -235,6 +252,79 @@ const int PIN_PWM_R1 = 9;
 const int PIN_PWM_R2 = 10;
 const int PIN_LED_LI = 4;
 const int PIN_LED_RI = 7;
+//-------------------------RTR_520----------------------//
+#elif (OPENBOT == RTR_520)
```

Comment:
I need to redefine the min and max functions to get it to build: 
```
#define max(a, b)                   \
    (                               \
        {                           \
            __typeof__(a) _a = (a); \
            __typeof__(b) _b = (b); \
            _a > _b ? _a : _b;      \
        })

#define min(a, b)                   \
    (                               \
        {                           \
            __typeof__(a) _a = (a); \
            __typeof__(b) _b = (b); \
            _a < _b ? _a : _b;      \
        })
```

Reply:
This should be fixed now with the fix below.